### DISCUSSION TITLE
Discussion Topics for Sep 29, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#2388] [Show and tell]
Discussion Topics for Sep 29, 2021 Engineering Sync

**Comment by carlosfelix2** [comment]
1. Database based lookup tables for Sender Automation team - @carlosfelix2

**Comment by whytheplatypus** [comment]
the joys of git repository policies when working in teams. (git practice, history, etc.)

North stars:
- A feeling of trust in the current state of the main branch
- A history that communicates changes and allows for recover operations (etc.)

**Comment by jorg3lopez** [comment]
Ktlint pre-commit githook.
- Checks for code violations
- Auto-format option 
- Some code violations can't be auto-formatted

**Comment by carlosfelix2** [comment]
@RickHawesUSDS Release to go out discussion.
